tests/packages.yml

```diff
@@ -0,0 +1,23 @@
+---
+acquia/lightning:
```

Comment:
@TravisCarden: This file is also referenced in .travis.yml as ORCA_PACKAGES_CONFIG_ALTER. Will these changes have any ill side effects due to that?

Reply:
Actually, @phenaproxima, `tests/packages_alter.yml` is referenced by `ORCA_PACKAGES_CONFIG_ALTER`. Each file is used in only one case, so there should be problem.

Reply:
Ah, thanks! I missed that, appreciate you pointing that out. Carry on!